src/Prowlarr.Api.V1/Search/ReleaseResource.cs

```diff
@@ -36,6 +37,21 @@ public class ReleaseResource : RestResource
         public int? Seeders { get; set; }
         public int? Leechers { get; set; }
         public DownloadProtocol Protocol { get; set; }
+
+        public string FileName
+        {
+            get
+            {
+                var extension = "torrent";
```

Comment:
Magnets?

Reply:
They should fail similar to now when save is clicked in the UI, shouldn't be any change in behavior on that front.